Add hide_from_help_command option to commands (#169)
sedders123/phial: CHANGELOG.md

### Added
 - Added 'loopDelay' config option. Specifes how to long to sleep for in the main loop, this helps to prevent super high CPU usage while doing nothing. Defaults to 0.001

>  - Added `hide_from_help_command` option upon command creation to hide a command from the inbuilt help command.

Author: ```suggestion
 - Added `hide_from_help_command` argument to command creation. This hides the command from the list generated by inbuilt help command.
```
